sale_blanket_order/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+# Copyright 2018 Acsone
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Blanket Orders',
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'author': 'Acsone SA/NV,Odoo Community Association (OCA)',
+    'version': '10.0.1.0.0',
+    'website': 'https://github.com/OCA/sale-workflow',
+    'summary': "Blanket Orders",
+    'depends': [
+        'sale',
+        'web_action_conditionable',
+    ],
+    'data': [
```

Comment:
@andreparames Sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)